### PR TITLE
Models were added and the folder structure was changed>

### PR DESCRIPTION
**Pull request by mutasim77**

### CODE REVIEW

File: Quotely/urls.py [hunk 1]
```diff
@@ -1,21 +1,8 @@
...
 from django.contrib import admin
-from django.urls import path
+from django.urls import include, path
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-]
+    # path('admin/', admin.site.urls),
+    path('', include('base.urls')),
+    # path('', include('admin_panel.urls')),
```
Comment (mutasim77) [inline Quotely/urls.py:7]: i dont' think we need for admin panel, what do you think?

Comment (BushraAks) [reply Quotely/urls.py:7]: maybe admin can delete all quotes that other users posted? or no need?

**Comment by BushraAks** [comment]
go push it, no need admin i think

**Comment by mutasim77** [comment]
did you check file changes?

**Comment by BushraAks** [comment]
just checked it now, still looking

**Comment by BushraAks** [comment]
okayy got it

**Comment by mutasim77** [comment]
also if your code review done u can just approve and i'll push it

**Comment by BushraAks** [comment]
youu areee >>>>>

**Pull request merged**